src/Controllers/Content.php

```diff
@@ -86,6 +86,25 @@ public function index()
             $response->send();
         }elseif ( isset($_SESSION['lti']['issuer_client']) ) {
             $custom_email_id = ParamValidate::getKeyInCustomFields($_SESSION, 'person_email_primary');
+            if ( isset($_SESSION['lti_post']['placement']) && $_SESSION['lti_post']['placement'] == "testing_plc" ) {
```

Comment:
Fix the placement name.
Follow PSR-12 standards.

no spaces after and before the opening/closing braces.